.github/CODEOWNERS

```diff
@@ -5,7 +5,7 @@
 # the repo. Unless a later match takes precedence,
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
-#*       @global-owner1 @global-owner2
+#*       @global-owner1 @global-owner2   
```

Comment:
do we need to specify the approver here for this open-source repo, i think we will also specify the roles for the codeowners for those project who might fork this.

Reply:
@neoofklaytn  I was also thinking on how it should be. Should we give all the emails contributed to this repo ?

Reply:
few from their side would be enough, and the majority should be from ours. 3 approvals required for every merge to happen, wdyt?